setup.sh

```diff
@@ -250,24 +250,52 @@ create_organization() {
     fi
 
     # Update the secrets-open-source.toml file
-    echo -e "[skyvern]\nconfigs = [\n    {\"env\" = \"local\", \"host\" = \"http://0.0.0.0:8000/api/v1\", \"orgs\" = [{name=\"Skyvern\", cred=\"$api_token\"}]}\n]" > .streamlit/secrets.toml
+    echo -e "[skyvern]\nconfigs = [\n    {\"env\" = \"local\", \"host\" = \"http://0.0.0.0:8000/api/v1\", \"orgs\" = [{name=\"Skyvern-Open-Source\", cred=\"$api_token\"}]}\n]" > .streamlit/secrets.toml
```

Comment:
Can we keep this as `Skyvern`?

Reply:
sure, but I'm confused why we regist `org.name="Skyvern-Open-Source"` but get `orgs.name="Skyvern"` written in the sercretes.toml?

Reply:
The orgs.name inside streamlit is cosmetic -- it doesn't map to anything!

Reply:
aha, ok. I will revert this commit